api_gen/src/main.rs

```diff
@@ -32,10 +34,17 @@ impl bindgen::callbacks::ParseCallbacks for Renamer {
         // Remove ffx prefixes
         match item_info.kind {
             bindgen::callbacks::ItemKind::Module => None,
-            bindgen::callbacks::ItemKind::Type => item_info
-                .name
-                .strip_prefix("Ffx")
-                .or_else(|| item_info.name.strip_prefix("ffx")),
+            bindgen::callbacks::ItemKind::Type => {
+                if item_info.name == "ffxFunctions" {
+                    // Work around unlucky type name that would end up defining `Functions` twice in the bindings.
+                    Some(item_info.name)
+                } else {
+                    item_info
+                        .name
+                        .strip_prefix("Ffx")
+                        .or_else(|| item_info.name.strip_prefix("ffx"))
+                }
+            }
```

Comment:
@MarijnS95 I don't know if this is the correct workaround, but `ffx` would be stripped in this case resulting in duplicate definition of `Functions`. Is there a better way to handle this?